src/transformer.ts

```diff
@@ -98,7 +98,23 @@ module ts2 {
 
 }
 
+/**
+ * Configuration settings for the transformer.
+ */
+export interface IConfig {
+  /** List of decorator names that mark classes to attach runtime data. */
+  decoratorNames?: string[]
+}
+
+class TransformConfig {
+  decoratorNames: string[] = ['Reflective'];
+
+  applyConfig(config: IConfig) {
+    this.decoratorNames = config.decoratorNames || this.decoratorNames;
+  }
+}
 
+const transformConfig: TransformConfig = new TransformConfig();
```

Comment:
I put the configuration at the module scope for now but can move this if you have a different place you would like it.